web/src/components/predictions-components/multiple-category-choice.js

```diff
@@ -0,0 +1,132 @@
+        };
+      }
+    });
+    this.setState({ values });
+  }
+
+  render() {
+    return (
+      <form onSubmit={event => this.afterSubmission(event)}>
+        {this.props.choices && this.props.choices.map((choice, index) =>
+          <Box>
+            <Label>
+              {choice.name}: {this.state.values[index].current.toFixed(1)}
+            </Label>
+            <Box mt={1} mx={3}>
+              <Range
+                name={choice.name}
+                id={index}
+                step={STEP}
+                max={this.state.values[index].max}
+                min={0}
+                values={[this.state.values[index].current]}
+                onChange={values => this.handleChange(values[0], index)}
+                renderTrack={({ props, children }) => (
+                  <div
+                    onMouseDown={props.onMouseDown}
+                    onTouchStart={props.onTouchStart}
+                    style={{
+                      ...props.style,
+                      height: '5rem',
+                      display: 'flex',
+                      width: '100%'
+                    }}
+                  >
+                    <div
+                      ref={props.ref}
+                      style={{
+                        height: '0.3rem',
+                        width: '100%',
+                        borderRadius: '0.3rem',
+                        background: getTrackBackground({
+                          values: [this.state.values[index].current],
+                          colors: ['#C63F3F', '#ccc'],
+                          min: 0,
+                          max: this.state.values[index].max
+                        }),
+                        alignSelf: 'center'
+                      }}
+                    >
+                      {children}
+                    </div>
+                  </div>
+                )}
+                renderThumb={({ props }) => (
+                  <div
+                    {...props}
+                    style={{
+                      ...props.style,
+                      height: '0.8rem',
+                      width: '0.8rem',
+                      borderRadius: '0.3rem',
+                      backgroundColor: '#FFF',
+                      display: 'flex',
+                      justifyContent: 'center',
+                      alignItems: 'center',
+                      boxShadow: '0px 0.15rem 0.45rem #AAA'
+                    }}
+                  >
+                    <div
+                      style={{
+                        position: 'absolute',
+                        top: '-2rem',
+                        color: '#fff',
+                        fontWeight: 'bold',
+                        fontSize: '1rem',
+                        fontFamily: 'Arial,Helvetica Neue,Helvetica,sans-serif',
+                        padding: '0.3rem',
+                        borderRadius: '0.3rem',
+                        backgroundColor: '#2F2F2F'
```

Comment:
Make sure to run `prettier --write "**/*.js"` on the file so it standardizes these quotes